### DISCUSSION TITLE
dotnet 6 support

### DISCUSSION THREAD

**Discussion by rofr** [dotnet/MQTTnet#1373] [General]
Hi everybody,

Has anyone tested with dotnet 6 yet

Regards,
Robert

**Comment by behroozbc** [comment]
Hi 
Yes I tested, its okay 
Those projects you can see and test yourself 
https://github.com/behroozbc/SimpleMQTTClient
https://github.com/behroozbc/SimpleMQTTBroker

**Reply by rofr** [reply]
That's good to know, thank you! We are considering building something and I will get back with feedback if I find any issues. Cheers!
